server/routes/ad.ts

```diff
@@ -460,8 +523,8 @@ export default class AdService {
         mustQueries.push({
           query_string: {
             fields: ['indices'],
-            default_operator: 'OR',
-            query: `*${indices.trim().split(' ').join('* *')}*`,
+            default_operator: 'AND',
```

Comment:
Why need to change from OR to AND here ?

Reply:
This is related to returning correct results when an `indices` search field is passed to the `getDetectors()` function. Before, there was no calls that were actually utilizing this field so the bug was ignored, but it is now used in the create detector page when finding existing detectors based on the selected index.. Note that the above `search` field uses the correct `AND` operator as well.

Reply:
Thanks for fixing this old bug